dashboard-all-the-things/src/app/dashboards/external-url/safe.pipe.ts

```diff
@@ -0,0 +1,13 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { DomSanitizer } from '@angular/platform-browser';
+
+@Pipe({ name: 'safe' })
+export class SafePipe implements PipeTransform {
+
+  constructor(private sanitizer: DomSanitizer) {}
+
+  transform(url) {
+    return this.sanitizer.bypassSecurityTrustResourceUrl(url);
```

Comment:
This shouldn't be a certificate issue, more of an Angular block over injecting DOM.